example/passthrough_hp.cc

```diff
@@ -747,15 +747,18 @@ static void do_readdir(fuse_req_t req, fuse_ino_t ino, size_t size,
             break; // End of stream
         }
         d->offset = entry->d_off;
-        if (is_dot_or_dotdot(entry->d_name))
-            continue;
 
         fuse_entry_param e{};
         size_t entsize;
         if (plus) {
-            err = do_lookup(ino, entry->d_name, &e);
-            if (err)
-                goto error;
+            if (is_dot_or_dotdot(entry->d_name)) {
+		e.attr.st_ino = entry->d_ino;
+		e.attr.st_mode = entry->d_type << 12;
```

Comment:
Would you mind to add a comment that fuse kernel does not refresh attributes and with that also does not increase nlookup? Without that comment I wasn't sure and had to look it up. Something like
/* fuse kernel ignores attributes for these and also does not increase lookup count */

Reply:
Absolutely, I will add that in right now.